### DISCUSSION TITLE
Improvements to package-lock

### DISCUSSION THREAD

**Discussion by dheeraj1447** [npm/feedback#426] [CLI] [answered]
Can the package-lock file be improved? I'm not sure why is the `resolved` property even needed? Also, when I proxy to npm using another registry the resolved URL doesn't get updated to the actual ones.

**Comment by MylesBorins** [comment, marked as answer]
hey @dheeraj1447 thanks for opening this discussion.

If you would like to propose concrete changes to the package lock format the next step will be to open an issue over at https://github.com/npm/rfcs to kick off a discussion
